generated/java/google-cloud-error-reporting-v1beta1/src/main/java/com/google/cloud/errorreporting/spi/v1beta1/ErrorGroupServiceApi.java

```diff
@@ -180,7 +201,6 @@ public final ErrorGroupServiceSettings getSettings() {
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
   public final ErrorGroup getGroup(String groupName) {
-    GROUP_PATH_TEMPLATE.validate(groupName, "getGroup");
```

Comment:
All of the validations are gone, because we have not set the resource_name_treatment in any gapic configs.

Do we want this? Or should the default resource_name_treatment be different?